Software/facial_recognition/recognize.py

```diff
@@ -30,8 +31,27 @@ def recognize(classifier, trainer):
     # to its index in the array. Ex) Ozayr -> ID 0
     names = ["Ozayr"] 
 
+    # Check if the user has up to 3 cameras
+    def returnCameraIndexes():
+        index = 0
+        arr = []
+        while index < 3:
+            cap = cv2.VideoCapture(index)
+            if cap.isOpened() == True:
+                arr.append(index)
+            cap.release()
+            index += 1
+        cam_id = int(input(f"{len(arr)} cameras available. Enter a camera ID {arr}: "))
+        return cam_id
+
+    # Prompt user for camera if not given
+    if camera:
+        selection = int(camera)
+    else:
+        selection = returnCameraIndexes()
```

Comment:
instead of prompting user for input, I think we should we require a camera and throw an error or have a default one.

Reply:
The user doesn't get prompted if the camera is specified via command line. A camera is required and OpenCV will throw an error for us if one isn't detected. The reason I added this is that I have 3 cameras hooked up to my computer so while testing the scripts, it was annoying if the wrong one was chosen or having to change the code if I wanted to change the camera.